legion/engine/physics/physx/systems/physx_physics_system.cpp

```diff
@@ -153,30 +293,29 @@ namespace legion::physics
         PS::cooking->release();
         PS::dispatcher->release();
         PS::physxSDK->release();
-
-        if (PS::pvd)
-        {
-            PxPvdTransport* transport = PS::pvd->getTransport();
-            PS::pvd->release();
-            PS::pvd = nullptr;
-            transport->release();
-        }
-
+        PS::debugger.release();
```

Comment:
does it need to be released if it wasn't initialised?

Reply:
the release function does check if its initialized and won't do anything if its not